mods/ts/sequences/infantry.yaml

```
@@ -109,18 +103,13 @@ e3:
 	icon:
 		Filename: e4icon.shp
 
-engineer.gdi:
+engineer:
 	Inherits: ^BasicInfantry
 	Defaults:
 		Filename: engineer.shp
 	icon:
```

```suggestion
	icon.gdi:
```

Shouldn't this be have gdi postfix then? Same with the rest of the sequences?

Though to enable that we'd have to update `CheckSequences` lint test. It's currently unaware of postfixes